src/main/kotlin/org/sd_jwt/SdJwt.kt

```diff
@@ -155,8 +148,13 @@ fun createSdClaims(
 }
 
 /**
- * This method creates a SD-JWT credential that contains the claims
- * passed to the method and is signed with the issuer's key.
+ * [LEGACY METHOD]
```

Comment:
The createCredential method that processes data classes should not be a legacy method. I do not plan to remove this method. I would like you to change the issuerKey: JWK parameter to signer: SdJwtSigner. I know this is a breaking change but there was no official release yet so I think this is okay. Other than that the PR looks good and is ready to merge I think.

Reply:
Alright thanks, I have removed the "old" method such that `createCredential` always expects a signer.

Reply:
I think there was a misunderstanding. I do want to keep two createCredential methods. One is to create a credential from a JSONObject using a signer and the other one creates a credential from a data class. I only wanted you to change the legacy method to use a signer and remove the legacy comment.

Reply:
Ah okay, I understand; should be fixed now!

Reply:
Ah okay, I understand; should be fixed now! Please have another look.